build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -4,7 +4,10 @@
   <UsingTask TaskName="GenerateBuildVersionInfo" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="CheckPrereqs" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="SetEnvVar" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="DotNetBuild" AssemblyFile="$(CLIBuildDll)" />
```

Comment:
Let's do this in a separate PR. For now, I've put them in the same place as other tasks coming from CLIBuildDll. Let's rationalize our Init/build.proj/Prepare story in one fell swoop.